dependencies/openssl.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please delete it, it's shipped by runtime.

Reply:
ok